tensorflow/python/ops/gradients.py

```diff
@@ -829,3 +829,45 @@ def _hessian_vector_product(ys, xs, v):
 
   # Second backprop
   return gradients(elemwise_products, xs)
+
+
+def hessian(ys, x, name="hessian", colocate_gradients_with_ops=False, 
+            gate_gradients=False, aggregation_method=None):
+  """Constructs the Hessian of sum of `ys` with respect to `x`.
+
+  `hessian()` adds ops to the graph to output the Hessian of `ys` with 
+  respect to `x` and returns a `Tensor`.
+
+  Args:
+    ys: A `Tensor` or list of tensors to be differentiated.
+    x: A `Tensor` to be used for differentiation.
```

Comment:
I will make changes such that `hessian` returns a list of tensors of the same length of `xs` for consistency with `gradients` and will rename to `hessians`.